lib/image-stream/supported.js

```diff
@@ -16,6 +16,13 @@
 
 'use strict';
 
+/**
+ * Supported filename extensions
```

Comment:
Can we use `@summary` for the title, and `@description` for the long comment? For example:

```js
/**
 * @summary Supported filename extensions
 * @description
 * NOTE: Extensions with type: 'compressed' will be stripped
 * ...
 */
```